ganga/GangaCore/Lib/Localhost/LocalHostExec_batch.py.template

```diff
@@ -8,7 +8,7 @@ from pathlib import Path
 from os.path import dirname, join, isdir
 
 def mp_worker(name):
-    print(f'START: {name} '+ time.strftime('%a %b %d %H:%M:%S %Y',time.gmtime(time.time())))
+    # print(f'START: {name} '+ time.strftime('%a %b %d %H:%M:%S %Y',time.gmtime(time.time())))
```

Comment:
Some different problem. The code itself is clearly valid python.

Reply:
do you get a traceback?

Reply:
This doesn't generate a traceback, but the print-out is sent to the Ganga prompt, rather than to the log file.  Given the redefinition of `sys.stdout` in [/ganga/GangaCore/Lib/Localhost/LocalHostExec_batch.py.template](https://github.com/kh296/ganga/blob/9df995563543e7dc9da3f5ff8e61268fbfda2d1f/ganga/GangaCore/Lib/Localhost/LocalHostExec_batch.py.template#L36), I'm not sure why the output isn't being captured.